### PR TITLE
Add option for a background image between videos

### PR DESCRIPTION
**Pull request by emmanuelgeoffray**
I needed this option to fake a gapless change of video in a playlist.
All my videos finish with the same logo, so I used an image of this last frame, which is displayed between videos.

Folks could also use this feature to show a wallpaper between videos.

### CODE REVIEW

File: Adafruit_Video_Looper/video_looper.py [hunk 1]
```diff
@@ -96,6 +97,17 @@ def _load_file_reader(self):
         return importlib.import_module('.' + module, 'Adafruit_Video_Looper') \
             .create_file_reader(self._config)
 
+    def _load_bgimage(self):
```
Comment (tofuSCHNITZEL) [inline Adafruit_Video_Looper/video_looper.py]: I propose a more robust function:

    def _load_bgimage(self):
        """Load the configured background image and return an instance of it."""
        image = None
        if self._config.has_option('video_looper', 'bgimage'):
            image = self._config.get('video_looper', 'bgimage')
            if not image:
                self._print('Using ' + str(image) + ' as a background')
                image = pygame.image.load(image)
                image = pygame.transform.scale(image, self._size)
        return image

File: Adafruit_Video_Looper/video_looper.py [hunk 2]
```diff
@@ -136,6 +148,9 @@ def _build_playlist(self):
     def _blank_screen(self):
         """Render a blank screen filled with the background color."""
         self._screen.fill(self._bgcolor)
+        if self._bgimage is not None:
+          rect = self._bgimage.get_rect()
+	  self._screen.blit(self._bgimage, rect)
```
Comment (tofuSCHNITZEL) [inline Adafruit_Video_Looper/video_looper.py]: indentation issue
```suggestion
            self._screen.blit(self._bgimage, rect)
```

File: Adafruit_Video_Looper/video_looper.py [hunk 3]
```diff
@@ -136,6 +148,9 @@ def _build_playlist(self):
     def _blank_screen(self):
         """Render a blank screen filled with the background color."""
         self._screen.fill(self._bgcolor)
+        if self._bgimage is not None:
+          rect = self._bgimage.get_rect()
```
Comment (tofuSCHNITZEL) [inline Adafruit_Video_Looper/video_looper.py]:
```suggestion
            rect = self._bgimage.get_rect()
```

**Comment by tofuSCHNITZEL** [comment]
thank you @emmanuelgeoffray for the suggestion

**Comment by emmanuelgeoffray** [comment]
Thank you @tofuSCHNITZEL ! Glad the suggestion found a listener :)
have a good day!

**Pull request merged**